src/styles/main.scss

```diff
@@ -21,6 +21,42 @@ body {
   vertical-align: center;
   user-select: none;
 
+  &--moveRight {
+    transform: translate(254px, 0);
+    transition: transform 0.5s linear;
+    color: #fff;
```

Comment:
Move all repeatable colors to the CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)